shared/js/angularProjects.js

```diff
@@ -1,82 +1,83 @@
-  'use strict';
+/* global $, angular */
```

Comment:
If you want to dive into this, [browserify-shim](https://github.com/thlorenz/browserify-shim) provides features for handling this type of transitional situation. See [Expose global variables via `global:*`](https://github.com/thlorenz/browserify-shim#a-expose-global-variables-via-global). For bundling code within Dashboard, we could use a config to shim the modules it expects to be provided on the global like this:
```
"browserify-shim": {
  "jquery": "global:$",
  "angular": "global:angular"
}
```
then you use `require('jquery')` and `require('angular')` within the modules. we could then use the same JS code outside of dashboard's environment (e.g. in js unit tests, or other bundles), loading the packages from npm dependencies.